utils/index.ts

```diff
@@ -18,3 +19,21 @@ export const emailFormSchema = z.object({
     .min(100, "Message should be at least 100 characters long"),
   contact_info: z.string().nonempty("Contact information is required"),
 });
+
+export const reorderWorkProcesses = (
+  workProcesses: WorkProcessSymbolType[]
+): WorkProcessSymbolType[] => {
+  const desiredOrder = [
+    "Definition",
+    "Design",
+    "Planning",
+    "Development",
+    "Deployment",
+  ];
```

Comment:
Instead of manually creating a "desired Order," another approach would first be importing the hard-coded data and then using the `sort` method to render the desired order. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

Reply:
I've changed this to use the sort method instead. It's working well now

Reply:
Excellent job buddy!